packages/oas-validator/index.js

```diff
@@ -13,17 +13,16 @@ let ajv = require('ajv')({
     allErrors: true,
     verbose: true,
     jsonPointers: true,
-    patternGroups: true,
-    extendRefs: true // optional, current default is to 'fail', spec behaviour is to 'ignore'
+    extendRefs: true, // optional, current default is to 'fail', spec behaviour is to 'ignore'
+    schemaId: 'id'
```

Comment:
Following the instructions on using draft-04 per https://ajv.js.org/ 
![image](https://user-images.githubusercontent.com/10089668/88896629-772ed100-d28d-11ea-9d58-b52a15ea2a7e.png)